doc/ros_visualization/visualization_tutorial.rst

```diff
@@ -15,11 +15,23 @@ and you should now have a MoveIt! configuration for the PR2 that you
 can use.  This tutorial assumes the generated MoveIt! configuration
 package is called "pr2_moveit_config".
 
-Alternately, you can just install the pre-made MoveIt! configuration
-for the PR2 in the pr2_moveit_config ROS package.  To install it,
-run::
+Alternately, you can source the MoveIt! configuration
+for the PR2 in the pr2_moveit_config ROS package. If you don't yet
+have a workspace for this tutorial create one below. Otherwise continue 
+to sourcing::
 
-  sudo apt-get install ros-indigo-moveit-pr2
+  mkdir moveit_ws
+  cd moveit_ws
+  mkdir src
+  cd ..
```

Comment:
Can you change this to the same two-liner used in the [source install](http://moveit.ros.org/install/source/) instructions?

```
mkdir -p ~/ws_moveit/src
cd ~/ws_moveit/src
```